apps/docs/hooks/Math/useCeil/src/use-ceil.demo.tsx

```
@@ -0,0 +1,205 @@
+      if (!inputValue) {
+        throw new Error('Please enter a value');
+      }
+
+      if (type === 'bigint') {
+        if (inputValue.includes('.')) {
+          throw new Error('BigInt cannot have decimal points');
+        }
+        setValue(BigInt(inputValue));
+      } else {
+        const num = Number(inputValue);
+        if (isNaN(num)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)